homeassistant/components/yamaha/media_player.py

```diff
@@ -53,6 +55,8 @@
     {vol.Required(ATTR_ENABLED): cv.boolean, vol.Required(ATTR_PORT): cv.string}
 )
 
+SELECT_SCENE_SCHEMA = MEDIA_PLAYER_SCHEMA.extend({vol.Required(ATTR_SCENE): cv.string})
```

Comment:
Please inline the schemas when registering the service instead as in the docs example. Leave out the entity_id item.

Reply:
Nowhere, I added it for completeness ...